mods/d2k/rules/aircraft.yaml

```diff
@@ -52,16 +52,16 @@ frigate:
 		Name: Frigate
 		Description: Supply spacecraft
 	Plane:
-		ROT: 5
-		Speed: 350
+		ROT: 1
+		Speed: 126
 		RepairBuildings: repair_pad
 		RearmBuildings:
 		Repulsable: False
 	Health:
-		HP: 500
+		HP: 9990
```

Comment:
The frigate should be indestructable, so best remove `Health:` and `Armor:` completely, and `Targetable:` if it has it.